<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert3D User Interface (itksnap.org)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,1,0">
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Working Directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inWorkDir">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnChangeDir">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="window-new">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>C3D Command</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CommandEditor" name="teCommand">
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="teOutput">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Backspace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRun">
         <property name="text">
          <string>Execute</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionC3D_Manual"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommands"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences ...</string>
   </property>
  </action>
  <action name="actionC3D_Manual">
   <property name="text">
    <string>C3D Manual ...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandEditor</class>
   <extends>QTextEdit</extends>
   <header location="global">CommandEditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>teCommand</tabstop>
  <tabstop>btnRun</tabstop>
  <tabstop>inWorkDir</tabstop>
  <tabstop>btnChangeDir</tabstop>
  <tabstop>teOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
